ubcpi/static/js/src/d3-pibar.js

```diff
@@ -10,12 +10,14 @@ d3.custom = (d3.custom || {});
  *     {frequency: 0,  label: 'Option 5', class: 'ubcpibar'},
  * ]
  */
-d3.custom.barChart = function() {
+d3.custom.barChart = function(scope) {
     // Private Variables
     var chartWidth  = 750;
     var chartHeight = 250;
     var minTotalFrequency = 10;
 
+    if(scope.role == 'instructor' || scope.role == 'staff'){ minTotalFrequency = 1};
```

Comment:
Where does this `.role` attribute get defined.  I see some code below, where there's a `$scope` getting `.user_role` assigned to it, but not `.role`.